tests/tests_unit/test_data_classes/test_data_models/test_data_types.py

```diff
@@ -37,12 +37,28 @@ class TestPropertyType:
                 "container": {"space": "mySpace", "externalId": "myId", "type": "container"},
                 "list": True,
             },
+            {
+                "type": "enum",
+                "values": {
+                    "string": {
+                        "name": "string",
+                        "description": "Time series with string data points.",
+                    },
+                    "numeric": {
+                        "name": "numeric",
+                        "description": "Time series with double floating point data points.",
+                    },
+                },
+            },
         ],
+        ids=lambda d: d.get("type", "unknown"),
     )
     def test_load_dump(self, data: dict) -> None:
-        actual = PropertyType.load(data).dump(camel_case=True)
+        loaded = PropertyType.load(data)
+        dumped = loaded.dump(camel_case=True)
 
-        assert data == actual
+        assert isinstance(loaded, PropertyType)
+        assert data == dumped
```

Comment:
Can this test be removed? I see `PropertyType` inherits from `CogniteObject` which I believe is covered by your automatic tests

Reply:
No, the issue is that you want to check that you load a `PropertyType` and not an `UnknownCogniteObject`. 

Note that when you only check `data == dumped` a `UnknownCogniteObject` will pass the type. Changed the check to make this clearer.

Reply:
Right, makes sense! 👍